### DISCUSSION TITLE
start button

### DISCUSSION THREAD

**Discussion by Gavincode-cyber** [Cog-Creators/Red-DiscordBot#5358] [General] [answered]
what do i put for configure the run button

**Comment by yamikaitou** [comment, marked as answer]
I suspect you are using an Unsupported host. Please make sure you are attempting to install Red on a VM/VPS directly instead of a service like Heroku/Repl.it/Pterodactyl. If you have further issues, I invite you to join us in Discord at discord.gg/red
